tests/pytests/test_cert_framework_basic.py

```diff
@@ -53,6 +53,29 @@ def test_getmembers_of_cc_framework():
     for item in ccf_builtin_methods:
         print(' -', item[0], item[1])
 
+# ##############################################################################
+def test_cf_cc_trust_data_default_ctor():
+    """
+    Basic exerciser for an empty cc_trust_data() object.
+    """
+    cctd = libcf.new_cc_trust_data()
```

Comment:
Without the c++ fix, this single line will trip the seg-fault consistently on every new run.